cli/crates/cli/tests/graphql-directive/main.rs

```diff
@@ -150,6 +151,43 @@ async fn graphql_test_without_namespace() {
     );
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn test_header_forwarding() {
+    server::run(54302).await;
+
+    let mut env = Environment::init_async().await;
+    let client = start_grafbase(&mut env, schema(54302, false)).await;
+
+    insta::assert_yaml_snapshot!(
```

Comment:
What are those tests doing? I see 4 times the same query with different `id`, not sure to see how that's related to the headers. 🤔

Reply:
Copy & paste fail, will delete